examples/getting_started/3_Deep_dive_into_the_anatomy_of_quantum_circuits.ipynb

```diff
@@ -2,35 +2,37 @@
  "cells": [
```

Comment:
<p>Could we add the execution result (as the one before edit)? It makes it easier for the user to grasp the idea. Same comment to the following cells with missing output.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-braket-examples/pull/127/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>

Reply:
Can you point out the number of cell that was not executed? (not clear to me which cell you refer to) From the what I see here all cell are executed.